scripts/leaderboard_backend/database_helper.py

```diff
@@ -0,0 +1,61 @@
+# type: ignore
+import select
+from dataclasses import dataclass
+
+import pandas as pd
+import psycopg2
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+from sqlalchemy import MetaData, Table, create_engine, insert, text
+from sqlalchemy.orm import sessionmaker
+
+
+@dataclass
+class EvalResult:
+    model: str
+    single_choice: float
+    multiple_choice: float
+    word_gen: float
+
+
+class DatabaseHelper:
+    def __init__(self):
+        self.engine = create_engine(
+            "postgresql+psycopg2://{username}:{passwd}@{ip}:5432/{db}",
+            echo=True,
+        )
+        self.engine.connect()
+        conn_string = "dbname={db} user={username} password={passwd} port='5432' host={ip}"
```

Comment:
How do you specify username, password and ip?

Reply:
environment variables